webapp/src/Service/ImportExportService.php

```diff
@@ -251,6 +252,23 @@ public function importContestData(mixed $data, ?string &$errorMessage = null, st
         }
 
         $this->em->persist($contest);
+
+        $shadow_type = $data['shadow_type'] ?? null;
```

Comment:
We should document that you can do this whole thing

Reply:
Makes sense, although the only users using this are us IMO.

Reply:
Which I agree with but I don't want to check the code to see how the fields are called. Also, maybe we use a subkey?
```yaml
shadow:
  type: xxx
```
etc?

Also makes the code more readable.